packages/fxa-settings/src/models/reliers/base-relier.ts

```diff
@@ -107,7 +107,7 @@ export class BaseRelier implements ModelContextProvider, Relier {
   @bind([V.isString], T.snakeCase)
   entrypointVariation: string | undefined;
 
-  @bind([V.isString], T.snakeCase)
+  @bind([V.isBoolean], T.snakeCase)
```

Comment:
Not sure if I'm reading this right, but can this be a Boolean _and_ snake case?
This is currently the only non `V.isString`ed snake case; all the others are `@bind([V.isString], T.snakeCase)`).

Reply:
T.snakeCase applies to the key, the validations ([V.isString]) applies to the value. 

BTW, the arguments for bind are @bind([ListOfValidors], KeyTransformation), and the key transformation is optional. For example, @bind([V.isBoolean], T.snakeCase) on a field called entrypointVariant would indicate the key in the underlying context is called entrypoint_variant, and it has a value that can be parsed as a boolean.